### PR TITLE
schema org

### PR DESCRIPTION
**Pull request by ipatate**

### CODE REVIEW

**Comment by ipatate** [comment]
https://validator.schema.org/#url=https%3A%2F%2Fdeploy-preview-67--double-slash-website.netlify.app%2Fpodcasts%2Fnews-novembre-2022%2F

File: composables/useHeadPodcast.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
+const getImgPodcast = ({
```
Comment (xlanex6) [inline composables/useHeadPodcast.ts:1]: Nice refacto

File: pages/podcasts/[slug].vue [hunk 2]
```diff
@@ -48,68 +42,9 @@ if (!episode.value) {
...
-      content: `https://res.cloudinary.com/doubleslash/image/upload/co_rgb:a700ff,g_east,l_text:mono.otf_120_letter_spacing_-5:%23${episode.value.episodeNumber},x_54/co_rgb:a700ff,g_east,l_text:mono.otf_120_letter_spacing_-5:${episode.value.title},x_54,y_150,w_1000/v1597238012/FACEBOOK_-_OG_Card_RAW_eu5xdv.png`,
-    },
-    {
-      hid: 'twitter:image:alt',
-      property: 'twitter:image:alt',
-      content: `${episode.value.episodeNumber} - ${episode.value.title}`,
-    },
-  ],
-});
+useHeadPodcast({episode, path});
```
Comment (xlanex6) [inline pages/podcasts/[slug].vue:45]: Lean 
well done

**Review by xlanex6 — APPROVED** [review]
Well done

**Comment by xlanex6** [comment]
🚀

**Pull request merged**